File: CS/CustomDisplayText/Form1.cs
using System;
using System.Collections.Generic;
using System.ComponentModel;
using System.Data;
using System.Drawing;
using System.Text;
using System.Windows.Forms;

namespace CustomDisplayText {
    public partial class Form1 : Form {
        public Form1() {
            InitializeComponent();

            for(int i = 0; i < 10; i++)
                dataTable1.Rows.Add(new object[] { i });
            textEdit2.EditValue = 3;
        }
    }
}

File: VB/CustomDisplayText/Form1.vb
Imports System
Imports System.Collections.Generic
Imports System.ComponentModel
Imports System.Data
Imports System.Drawing
Imports System.Text
Imports System.Windows.Forms

Namespace CustomDisplayText
	Partial Public Class Form1
		Inherits Form

		Public Sub New()
			InitializeComponent()

			For i As Integer = 0 To 9
				dataTable1.Rows.Add(New Object() { i })
			Next i
			textEdit2.EditValue = 3
		End Sub
	End Class
End Namespace